test/gtest/ucp/test_ucp_tag_match.cc

```diff
@@ -37,9 +37,8 @@ class test_ucp_tag_match : public test_ucp_tag {
         if (use_proto()) {
             modify_config("PROTO_ENABLE", "y");
             modify_config("MAX_EAGER_LANES", "2");
-        } else if (RUNNING_ON_VALGRIND && m_ucp_config->ctx.proto_enable) {
-            UCS_TEST_SKIP_R("FIXME: skip forced UCX_PROTO_ENABLE=y because "
-                            "it does not completely support HWTM");
+        } else if (RUNNING_ON_VALGRIND) {
```

Comment:
1. seems condition of "else" is changed
2. do we want to skip protov2 only with valgrind?

Reply:
why need to skip protov2 only on valgrind in some tests?

Reply:
see description to https://github.com/openucx/ucx/pull/8756